socialhome/templates/base.html

```diff
@@ -50,9 +50,11 @@
                               <a class="nav-link" href="{% url 'account_logout' %}">{% trans "Logout" %}</a>
                           </li>
                       {% else %}
+                          {% if is_open_for_signup %}
```

Comment:
No, I don't think it works. That's the thing: I don't know where to inject the variable.